src/app.tsx

```diff
@@ -1,11 +1,53 @@
+import { lazy } from 'react';
 import { Route, Routes } from 'react-router-dom';
 
 import { AppRoute } from '@/common/enums/enums.js';
 
+import { LayoutContainer } from './Components/layout-container.jsx';
+
+const Visits = lazy(() =>
+  import('./Components/visits.js').then((module) => {
+    return {
+      ...module,
+      default: module.Visits,
+    };
+  }),
+);
```

Comment:
```suggestion
const Visits = lazy(() => 
  import('./Components/visits.js')
  .then((module) => ({default: module.Visits})),
);
```
And create a helper function to make the code less duplicated and more simple, so it can be used like usual React's lazy with default exports.